src/index.ts

```diff
@@ -44,11 +73,11 @@ export default class SceneWriter {
   private writeComponent(component: DCL.ObservableComponent) {
     // Get class name
     let constructor
-    const typesArray = Object.keys(typesMap.exports).filter(
-      n => typesMap.exports[n].kind === 'class' && n !== 'Shape' && n !== 'ObservableComponent'
+    const typesArray = Object.keys(this.map.exports).filter(
```

Comment:
Comment, what is this?

Reply:
types array are all the "types" map object that are classes, but not shape or observable component. So every time the scene writer wants to know of which class is an object instance of we have to just look in here